test/3_cmake_targets.vader

```diff
@@ -9,7 +9,7 @@ Before:
 
     function! RemoveCMakeDirs()
         if has('win32')
-            for l:value in ['cmake-build-release', 'cmake-build-debug', 'cmake-build-relwithdebinfo', 'cmake-build-minsizerel', 'cmake-build', 'build_test']
+            for l:value in ['cmake-build-Release', 'cmake-build-Debug', 'cmake-build-relwithdebinfo', 'cmake-build-minsizerel', 'cmake-build', 'build_test']
```

Comment:
You have done backward incompatible changes when you removed `toLower()` call for build type.

In this line you also need to fix `relwithdebinfo` and `minsizerel` in this case.